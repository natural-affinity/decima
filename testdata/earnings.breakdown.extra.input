decima -e 25 --breakdown 100
