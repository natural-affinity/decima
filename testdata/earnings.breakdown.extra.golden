You owe: $35.00

Takehome: $65.00
Earnings: $100.00
 Percent: 10.00%
   Tithe: $10.00
   Extra: $25.00
